aws/resource_aws_emr_instance_group_test.go

```diff
@@ -29,6 +29,22 @@ func TestAccAWSEMRInstanceGroup_basic(t *testing.T) {
 	})
 }
 
+func TestAccAWSEMRInstanceGroup_spot(t *testing.T) {
+	var ig emr.InstanceGroup
+	rInt := acctest.RandInt()
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckAWSEmrInstanceGroupDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAWSEmrInstanceGroupConfig_bid(rInt),
+				Check:  testAccCheckAWSEmrInstanceGroupExists("aws_emr_instance_group.task", &ig),
```

Comment:
To add multiple checks for this configuration, we should definitely use this approach:

```go
        Check: resource.ComposeAggregateTestCheckFunc(
            testAccCheckAWSEmrInstanceGroupExists("aws_emr_instance_group.task", &ig),
            resource.TestCheckResourceAttr("aws_emr_instance_group.task", "bid_price", "0.10"),
        ),
```

This would allow to check that the EmrInstanceGroupExists and that the bid_price is well set. This depends on the `d.Set("bid_price", group.BidPrice)` instruction I talked about :)